extension/src/prototypes/view/ui-containers/EnvironmentSelect.tsx

```diff
@@ -239,6 +240,10 @@ export const EnvironmentSelect: FC = () => {
         {...bindTrigger(popupState)}>
         <MapIcon />
       </AppIconButton>
+      {/* TODO: Implment the click, popup and pass props */}
```

Comment:
Could you remove this?

Reply:
Just did, before you commented :laughing: